solutions/cses-1625.mdx

```diff
@@ -0,0 +1,349 @@
+---
+id: cses-1625
+source: CSES
+title: Grid Paths
+author: Vivian Han
+---
+
+## Explanation
+
+Adapted from https://usaco.guide/CPH.pdf#page=61
+
+<Warning title="Run times">
+Please note: the running times and number of recursive calls listed are for a separate but very similar problem (# paths from upper left to *lower right* corner on a 7x7 grid). 
```

Comment:
```suggestion
Please note: the running times and number of recursive calls listed are for a separate but very similar problem (finding the number of paths from the upper-left to *lower-right* corner of a 7x7 grid). 
```
this problem is from cph, right? maybe link it?

Reply:
this problem is from cses

Reply:
the problem mentioned in the warning?

Reply:
I linked the CPH page at the top of the solution, and that seems to be the only place where the problem is mentioned...

Reply:
sure, that looks good

can you commit the suggestion?